tests/e2e/indexing.spec.ts

```diff
@@ -0,0 +1,82 @@
+import { expect } from "chai";
+import axios from "axios"
+
+
+const NUMBER_OF_TRANSFERS = 32; 
+
+describe("E2E tests - EVM", async function() {
+
+    let evmTransfers: Array<any> 
+
+    before(async() =>{
+        const response = await axios.get("http://localhost:8000/api/transfers?page=1&limit=100")
+        expect(response.status).to.be.deep.equal(200)
+
+        const transfers = response.data as Array<any>
+        expect(transfers.length).to.be.deep.equal(NUMBER_OF_TRANSFERS)
+
+        evmTransfers = transfers.filter((transfer) => 
+            transfer.fromDomain.name == "Ethereum 1" && transfer.toDomain.name == "evm2"
+            || transfer.fromDomain.name == "evm2" && transfer.toDomain.name == "Ethereum 1") 
+        })
+    
+    it("Expect transfer count matches", async () => {
+
+        expect(evmTransfers.filter((transfer) => 
+            transfer.fromDomain.name == "Ethereum 1" && transfer.toDomain.name == "evm2"
+        ).length).to.be.deep.equal(57);  //number should be checked
+
+        /* --- There are currently no evm2 => Ethereum 1 transfers --- 
```

Comment:
Should we remove this commented code?

Same question for below commented code

@mpetrun5